W-20284621 Update Getting Started (#17)
mulesoft/docs-mulesoft-mcp-server: modules/ROOT/pages/reference-mcp-tools.adoc

NOTE: Some MuleSoft MCP Server tools are available only in the Anypoint Code Builder IDE or Connector Builder. 

> See xref:getting-started.adoc#create-a-connected-app-and-add-oauth-permission-scopes[Add OAuth Permission Scopes] for details on which tools require specific permission scopes.

Reviewer: Our editors would say "information about which tools"...

> See xref:getting-started.adoc#create-a-connected-app-and-add-oauth-permission-scopes[Add OAuth Permission Scopes] for details on which tools require specific permission scopes.

Author: ```suggestion
See xref:getting-started.adoc#create-a-connected-app-and-add-oauth-permission-scopes[Add OAuth Permission Scopes] for information about which tools require specific permission scopes.
```
